test-kit/components/slider-driver.ts

```diff
@@ -0,0 +1,109 @@
+import * as keycode from 'keycode';
+import {DriverBase, simulate} from 'test-drive-react';
+import {ContextProvider, Slider} from '../../src';
+import WindowStub from '../../test/stubs/window.stub';
+import {simulateMouseEvent, simulateTouchEvent} from '../../test/utils';
+
+// TODO reuse
```

Comment:
explain what does it means

Reply:
just forgot to remove this. there are not "TODO" in this PR